src/surabhi/assignment10/Calculator.java

```diff
@@ -0,0 +1,42 @@
+package surabhi.assignment10;
+
+public class Calculator {
+
+	int add(int num1, int num2) {
+		return num1+num2; 
+	}
+	
+	int sub(int num1, int num2) {
```

Comment:
I wanted positive difference between two numbers. Hence used condition to perform subtraction

Reply:
Okay